cli/remote_build.go

```diff
@@ -187,87 +186,84 @@ func (p *RemoteCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}
-		log.Errorf("Couldn't find HEAD commit: %v", err)
+		log.Errorf(ctx, "Couldn't find HEAD commit: %v", err)
 		return subcommands.ExitFailure
 	}
 	headCommit, err := workRepo.CommitObject(head.Hash())
 	if err != nil {
-		log.Errorf("Couldn't find HEAD commit: %v", err)
+		log.Errorf(ctx, "Couldn't find HEAD commit: %v", err)
 		return subcommands.ExitFailure
 	}
 	ancestorCommit, err := workRepo.CommitObject(ancestorRef)
 	if err != nil {
-		log.Errorf("Couldn't find merge-base commit: %v", err)
+		log.Errorf(ctx, "Couldn't find merge-base commit: %v", err)
 		return subcommands.ExitFailure
 	}
 
 	// Show feedback: end of bootstrap
 	endTime := time.Now()
 	bootTime := endTime.Sub(startTime)
-	log.Infof("Bootstrap finished at %s, taking %s", endTime.Format(TIME_FORMAT), bootTime.Truncate(time.Millisecond))
+	log.Infof(ctx, "Bootstrap finished at %s, taking %s", endTime.Format(TIME_FORMAT), bootTime.Truncate(time.Millisecond))
 
 	// Process patches
 	startTime = time.Now()
 	pGenerationChan := make(chan bool)
 	if p.committed && headCommit.Hash.String() != p.baseCommit {
-		log.Infof("Generating patch for %d commits...", commitCount)
+		log.Infof(ctx, "Generating patch for %d commits...", commitCount)
 
 		patch, err := ancestorCommit.Patch(headCommit)
 		if err != nil {
-			log.Errorf("Patch generation failed: %v", err)
+			log.Errorf(ctx, "Patch generation failed: %v", err)
 			return subcommands.ExitFailure
 		}
 		// This is where the patch is actually generated see #278
 		go func(ch chan<- bool) {
-			log.Debug("Starting the actual patch generation...")
+			log.Debugf(ctx, "Starting the actual patch generation...")
 			p.patchData = []byte(patch.String())
-			log.Debug("Patch generation finished, only committed changes")
+			log.Debugf(ctx, "Patch generation finished, only committed changes")
 			ch <- true
 		}(pGenerationChan)
 	} else if !p.committed {
 		// Apply changes that weren't committed yet
 		worktree, err := workRepo.Worktree() // current worktree
 		if err != nil {
-			log.Errorf("Couldn't get current worktree: %v", err)
+			log.Errorf(ctx, "Couldn't get current worktree: %v", err)
 			return subcommands.ExitFailure
 		}
 
-		log.Info("Generating patch for local changes...")
+		log.Infof(ctx, "Generating patch for local changes...")
 
-		log.Debug("Start backing up the worktree-save")
+		log.Debugf(ctx, "Start backing up the worktree-save")
 		saver, err := types.NewWorktreeSave(targetPackage.Path, headCommit.Hash.String(), p.backupWorktree)
 		if err != nil {
-			log.Errorf("%s", err)
+			log.Errorf(ctx, "%s", err)
 		}
 
 		// TODO When go-git worktree.Status() get faster use this instead:
 		// There's also the problem detecting CRLF in Windows text/source files
 		//if err = worktree.AddGlob("."); err != nil {
-		if skippedBinaries, err := p.traverseChanges(worktree, saver); err != nil {
-			log.Error(err)
+		if skippedBinaries, err := p.traverseChanges(ctx, worktree, saver); err != nil {
+			log.Errorf(ctx, "%v", err)
```

Comment:
On line 239 we use `%s`, here we use `%v`, do we want to make these consistent?

Reply:
Done.